TSOClient/tso.simantics/Model/VMObjectQueries.cs

```diff
@@ -156,6 +187,14 @@ public void NewObject(VMEntity obj)
             }
             VM.AddToObjList(list, obj);
             RegisterCategory(obj, obj.GetValue(VMStackObjectVariable.Category));
+            if(obj.SemiGlobal != null)
+            {
+                if(obj.SemiGlobal.Iff.Filename != null)   //sanity check
```

Comment:
`if (`
If needs space

Also, what's with all the spaces before `//sanity check`?

Reply:
Not quite sure why I'd put three spaces there, but will fix it asap.

Reply:
Solved in fc27d4be1bda305e6d4dc490743be9e8b6c21285